packages/cli/register/src/ir-to-fdr-converter/convertTypeShape.ts

```diff
@@ -55,6 +55,9 @@ export function convertTypeShape(irType: Ir.types.Type): FdrCjsSdk.api.v1.regist
                     return {
                         description: variant.docs ?? undefined,
                         discriminantValue: variant.discriminantValue.wireValue,
+                        displayName: variant.displayName,
+                        availability:
+                            variant.availability != null ? convertIrAvailability(variant.availability) : undefined,
```

Comment:
you'll need to update the IR changelog + IR version with this change. 
- https://github.com/fern-api/fern/blob/main/packages/ir-sdk/fern/apis/ir-types-latest/changelog/CHANGELOG.md
- https://github.com/fern-api/fern/blob/main/packages/ir-sdk/fern/apis/ir-types-latest/VERSION